src/components/feed-entry.vue

```diff
@@ -83,4 +84,13 @@ defineProps({
 
   .markdown-view > p:last-child { margin: 0 0 0px; }
 }
+
+// Container of multiple FeedEntry components
+.feed-entry-group {
```

Comment:
I'm not sure that the `FeedEntry` component is exactly the right place for these styles. Maybe one day `.feed-entry-group` becomes its own component.